joomla/joomla-cms
# Update en-GB.com_content.ini

IGNORE THIS FOR NOW.  THERE NEEDS TO BE AN UPDATE TO ANOTHER FILE BUT IT SEEMS TO HAVE DISAPPEARED.  I don't know how to unpull request, but I will make the changes when I get a second.  Need to stop for now.

Pull Request for Issue # .

### Summary of Changes

Language string for article intro image were the same as article full image and therefore a language override did not work.  Interestingly there was a language string available which I did originally try to override but I realised it wasn't working as the XML file was wrong.



### Testing Instructions
Go to language overrides and create one for COM_CONTENT_FLOAT_INTRO_LABEL which should be available already.

### Expected result
Updates the intro image float label


### Actual result
Intro image and fulltext image float label are identical


### Documentation Changes Required
I don't know if this effects backwards compatibility or other components.  I updated fulltext image language strings to have _FULL as _INTRO was used in the other case and I thought we should be consistent.
